src/mono/wasm/debugger/BrowserDebugProxy/Common/HelperExtensions.cs

```diff
@@ -5,13 +5,27 @@
 
 using System;
 using Newtonsoft.Json.Linq;
+using BrowserDebugProxy;
+using System.Collections.Generic;
+using System.Linq;
 
 namespace Microsoft.WebAssembly.Diagnostics;
 
 internal static class HelperExtensions
 {
     private const int MaxLogMessageLineLength = 65536;
     private static readonly bool TruncateLogMessages = string.IsNullOrEmpty(Environment.GetEnvironmentVariable("WASM_DONT_TRUNCATE_LOG_MESSAGES"));
+    private static Dictionary<ProxyInternalUseProperty, string> proxyInternalUsePropNames = new Dictionary<ProxyInternalUseProperty, string>() {
```

Comment:
Thank you, works fine. I converted `InternalFieldName` static fields to strings because I was not sure about the size of the whole class. It seems like an inception when each of `InternalFieldName` fields contains 8 static fields of the same size and on and on... Strings serve our purpose equally good.